src/app/api/file/route.ts

```diff
@@ -0,0 +1,130 @@
+import { NextResponse } from "next/server";
+import { File, FileResponse } from "./route.schema";
+import { prisma } from "@server/db/client";
+import { withSession } from "@server/decorator";
+import { createDriveService } from "@server/service";
+
+export const POST = withSession(async (request) => {
+  try {
```

Comment:
I think we can get rid of `try-catch`. It would make debugging easier if we see the full error printed.